.eslintrc.cjs

```diff
@@ -33,6 +33,7 @@ module.exports = {
         caughtErrorsIgnorePattern: "^_",
       },
     ],
+    "@typescript-eslint/no-unnecessary-condition": "off",
```

Comment:
The better way to fix the error without adding this rule is to change the type of the variable.